### PR TITLE
Fixes #1428 - Adds test for activity page

### LINKED ISSUE #1428: Add test for `/activity/user`
To finish up #1419 (fixes #1412 ), there needs to be one more title added to the array being tested in `test_rendering.py` - the title for `/activity/user`. This will require mocking the circumstances of the /activity template. More info [here](https://github.com/webcompat/webcompat.com/pull/1419#issuecomment-286418716).

### PR DESCRIPTION
**Pull request by karlcow**
this finalizes the previous commit around testing rendering of the title in pages.

### CODE REVIEW

**Comment by karlcow** [comment]
@miketaylr r?

File: tests/test_rendering.py [hunk 1]
```diff
@@ -20,16 +20,27 @@
                                'rv:31.0) Gecko/20100101 Firefox/31.0')}
 
 
-class TestURLs(unittest.TestCase):
+class TestURIContent(unittest.TestCase):
+    """Tests for the content once HTML remplates are rendered."""
```
Comment (miketaylr) [inline tests/test_rendering.py]: nit: templates

Comment (karlcow) [reply tests/test_rendering.py]: ahah cute. Must have been rambling about the templates.
Thanks.

**Comment by karlcow** [comment]
typo fixed. Let's merge per comment https://github.com/webcompat/webcompat.com/pull/1727#pullrequestreview-59262697

**Pull request merged**